tests/product/test_create_images.py

```diff
@@ -0,0 +1,98 @@
+from tests.no_hadoop_bare_image_provider import NoHadoopBareImageProvider
+from tests.product.base_product_case import BaseProductTestCase
+
+import unittest
+
+
+@attr('quarantine')
+class TestSetupImages(BaseProductTestCase):
+    def _get_expected_image_names(self, cluster_type):
+        if cluster_type == self.BARE_CLUSTER:
+            slave_image_name = 'teradatalabs/pa_test/nohadoop_bare_slave'
+            master_image_name = 'teradatalabs/pa_test/nohadoop_bare_master'
+            return [slave_image_name, master_image_name]
+        elif cluster_type == self.PA_ONLY_CLUSTER:
+            slave_image_name = 'teradatalabs/pa_test/nohadoop_pa_only_standalone_slave'
+            master_image_name = 'teradatalabs/pa_test/nohadoop_pa_only_standalone_master'
+            return [slave_image_name, master_image_name]
+        elif cluster_type == self.STANDALONE_PRESTO_CLUSTER:
+            slave_image_name = 'teradatalabs/pa_test/nohadoop_presto_slave'
+            master_image_name = 'teradatalabs/pa_test/nohadoop_presto_master'
+            return [slave_image_name, master_image_name]
+        elif cluster_type == self.PA_ONLY_YS_CLUSTER:
+            slave_image_name = 'teradatalabs/pa_test/nohadoop_pa_only_ys_slave'
+            master_image_name = 'teradatalabs/pa_test/nohadoop_pa_only_ys_master'
+            return [slave_image_name, master_image_name]
+        else:
+            return None
+
+    def _assert_expected_images_created(self, cluster_type):
+        docker_images = check_output(['docker', 'images'])
+        expected_images = self._get_expected_image_names(cluster_type)
+        for expected_image in expected_images:
+            assert expected_image in docker_images
+
+    def _setup_cluster_with_no_hadoop_provider_assert_images(self, cluster_type):
+        self.setup_cluster(NoHadoopBareImageProvider(),
+                           cluster_type)
+        self._assert_expected_images_created(cluster_type)
+
+    def test_setup_standalone_presto_images(self):
+        cluster_type = self.STANDALONE_PRESTO_CLUSTER
+        self._setup_cluster_with_no_hadoop_provider_assert_images(cluster_type)
+
+    def test_setup_standalone_pa_only_images(self):
+        cluster_type = self.PA_ONLY_CLUSTER
+        self._setup_cluster_with_no_hadoop_provider_assert_images(cluster_type)
+
+    def test_setup_standalone_bare_images(self):
+        cluster_type = self.BARE_CLUSTER
+        self._setup_cluster_with_no_hadoop_provider_assert_images(cluster_type)
+
+    def test_setup_yarn_slider_pa_only_images(self):
+        cluster_type = self.PA_ONLY_YS_CLUSTER
+        self._setup_cluster_with_no_hadoop_provider_assert_images(cluster_type)
+
+
+if __name__ == "__main__":
+    # Allow arguments to be quoted in a string or input as multiple arguments:
+    # python test_create_images.py "standalone_presto standalone_pa_only"
+    # python test_create_images.py standalone_presto standalone_pa_only
+    command_line_arguments = argv[1:]
```

Comment:
You should use the argparse module for argument parsing https://docs.python.org/2/library/argparse.html.